components/Converters/samples/TypeToObjectConverterSample.xaml

```diff
@@ -0,0 +1,22 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="ConvertersExperiment.Samples.TypeToObjectConverterSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:ConvertersExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <converters:TypeToObjectConverter x:Key="TypeToObjectConverter"
+                                          FalseValue="Red"
+                                          TrueValue="Green"
+                                          Type="x:String" />
+    </Page.Resources>
+
+    <TextBlock Foreground="{x:Bind MyTextStringValue, Mode=OneWay, Converter={StaticResource TypeToObjectConverter}}"
```

Comment:
This one didn't work for me? @Arlodotexe thoughts on a better example for this one?

Reply:
Oh, forgot to add that to the known issues.. yeah, having a valid scenario would be helpful to improve this sample!

Reply:
Think this is the main one left that's wonky. The original sample used the TextToolbar which kind of made sense. I could see maybe a shared generic DataTemplate type scenario being a case, but searching GitHub I didn't see any usages of this converter... hmmm

Reply:
Added a working example, albeit still not practical, good enough for now.